examples/analytics_specs/main.tf

```diff
@@ -1,5 +1,6 @@
 module "atlas_basic" {
-  source = "../../"
+  source = "terraform-mongodbatlas-modules/atlas-basic/mongodbatlas"
+  version = "0.0.1"
```

Comment:
curious if "latest" would work here?

Reply:
feels like latest does not work, however if i do not specify version at all (skip defining the attribute version) it works

<img width="453" alt="Screenshot 2024-07-25 at 10 08 54" src="https://github.com/user-attachments/assets/24b7d55b-1cc0-4c3f-b0af-e6846028d66e">